lib/primo/parameter_validatable.rb

```diff
@@ -0,0 +1,48 @@
+module Primo
+  module ParameterValidatable
```

Comment:
Yup, this makes a lot more sense as it's own module.  I've been wondering if there is a way we can move a class version of the validators method.  It might be useful.